src/components/containers/Panel.js

```diff
@@ -103,7 +105,18 @@ export default class Panel extends Component {
             hasOpen={hasOpen}
             onAction={onAction}
           />
-          {newChildren}
+          {requireTraces ? (
+            !hasTraces ? (
+              <PanelEmpty
+                heading={_("Looks like there aren't any traces defined yet.")}
+                message={<p>{_("Go to the 'Create' tab to define traces.")}</p>}
+              />
+            ) : (
+              newChildren
+            )
+          ) : (
+            newChildren
+          )}
```

Comment:
This can be simplified - `requireTraces && !hasTraces ? <PanelEmpty /> : newChildren`

But also, looks like `<PanelEmpty>` is only called here and in `<TraceAccordion>`, with the same translated strings both times - can we just do those once inside `<PanelEmpty>`?